geode/src/test/java/org/apache/calcite/adapter/geode/rel/GeodeBookstoreTest.java

```diff
@@ -413,71 +479,132 @@ public void testFilterWithNestedField() {
         .explainContains("PLAN=GeodeToEnumerableConverter\n"
             + "  GeodeProject(postalCode=[ITEM($3, 'postalCode')])\n"
             + "    GeodeFilter(condition=[>(ITEM($3, 'postalCode'), '0')])\n"
-            + "      GeodeTableScan(table=[[geode, BookCustomer]])\n");
+            + "      GeodeTableScan(table=[[geode, BookCustomer]])\n")
+        .queryContains(
+            GeodeAssertions.query("SELECT primaryAddress.postalCode AS postalCode "
+            + "FROM /BookCustomer WHERE primaryAddress.postalCode > '0'"));
   }
 
   @Test
-  public void testSqlSimple() throws SQLException {
+  public void testSqlSimple() {
     calciteAssert()
         .query("SELECT itemNumber FROM geode.BookMaster WHERE itemNumber > 123")
-        .runs();
+        .runs()
+        .queryContains(
+            GeodeAssertions.query("SELECT itemNumber AS itemNumber "
+            + "FROM /BookMaster WHERE itemNumber > 123"));
   }
 
   @Test
-  public void testSqlSingleNumberWhereFilter() throws SQLException {
+  public void testSqlSingleNumberWhereFilter() {
     calciteAssert().query("SELECT * FROM geode.BookMaster "
-        + "WHERE itemNumber = 123").runs();
+        + "WHERE itemNumber = 123")
+        .runs()
+        .queryContains(
+            GeodeAssertions.query("SELECT * FROM /BookMaster "
+            + "WHERE itemNumber = 123"));
   }
 
   @Test
-  public void testSqlDistinctSort() throws SQLException {
+  public void testSqlDistinctSort() {
     calciteAssert().query("SELECT DISTINCT itemNumber, author "
-        + "FROM geode.BookMaster ORDER BY itemNumber, author").runs();
+        + "FROM geode.BookMaster ORDER BY itemNumber, author")
+        .runs()
+        .queryContains(
+            GeodeAssertions.query("SELECT itemNumber AS "
+            + "itemNumber, author AS author FROM /BookMaster GROUP BY "
+            + "itemNumber, author ORDER BY itemNumber ASC, author ASC"));
   }
 
   @Test
-  public void testSqlDistinctSort2() throws SQLException {
+  public void testSqlDistinctSort2() {
     calciteAssert().query("SELECT itemNumber, author "
         + "FROM geode.BookMaster GROUP BY itemNumber, author ORDER BY itemNumber, "
-        + "author").runs();
+        + "author")
+        .runs()
+        .queryContains(
+            GeodeAssertions.query("SELECT itemNumber AS itemNumber, "
+            + "author AS author FROM /BookMaster GROUP BY itemNumber, "
+            + "author ORDER BY itemNumber ASC, author ASC"));
   }
 
   @Test
-  public void testSqlDistinctSort3() throws SQLException {
-    calciteAssert().query("SELECT DISTINCT * FROM geode.BookMaster").runs();
+  public void testSqlDistinctSort3() {
+    calciteAssert().query("SELECT DISTINCT * FROM geode.BookMaster")
+        .runs()
+        .queryContains(
+            GeodeAssertions.query("SELECT itemNumber AS itemNumber, "
```

Comment:
There is no `DISTINCT` keyword in OQL ?

Reply:
The DISTINCT keyword is existing code, I just added the query assertion